cligen_history.c

```diff
@@ -111,11 +118,11 @@ hist_save(char *p)
  */
 int
 hist_add(cligen_handle h,
-         char         *buf)
+         const char         *buf)
```

Comment:
Style: indendation error according to https://github.com/clicon/clixon/blob/master/CONTRIBUTING.md#function-declarations